### PR TITLE
PROD. Added Polygon.

### PR DESCRIPTION
**Pull request by AndreiNavoichyk**

### CODE REVIEW

**Review by karthikeyan-cere — APPROVED** [review]

File: config/chainbridge-runtime-config.cere.prod.js [hunk 1]
```diff
@@ -5,30 +5,30 @@ const cereTokenDecimals = 10;
...
+        decimals: cereTokenDecimals,
+        bridgeAddress: "",
+        erc20HandlerAddress: "",
+        rpcUrl:
+          "https://polygon-mainnet.infura.io/v3/0aca1499facc499bb195d2d437f78603",
+        type: "Ethereum",
+        nativeTokenSymbol: "MATIC",
+        tokens: [
+          {
+            address: "",
```
Comment (AndreiNavoichyk) [inline config/chainbridge-runtime-config.cere.prod.js]: Add address

File: config/chainbridge-runtime-config.cere.prod.js [hunk 2]
```diff
@@ -5,30 +5,30 @@ const cereTokenDecimals = 10;
...
-      //       isDoubleApproval: false,
-      //     },
-      //   ],
-      // },
+      {
+        chainId: 2,
+        networkId: 137,
+        name: "Polygon Mainnet",
+        decimals: cereTokenDecimals,
+        bridgeAddress: "",
```
Comment (AndreiNavoichyk) [inline config/chainbridge-runtime-config.cere.prod.js]: Add address

File: config/chainbridge-runtime-config.cere.prod.js [hunk 3]
```diff
@@ -5,30 +5,30 @@ const cereTokenDecimals = 10;
...
+        erc20HandlerAddress: "",
```
Comment (AndreiNavoichyk) [inline config/chainbridge-runtime-config.cere.prod.js]: Add address

File: config/chainbridge-runtime-config.cere.prod.js (hunk 1 above)
Comment (AndreiNavoichyk) [reply config/chainbridge-runtime-config.cere.prod.js]: Added

**Pull request merged**